Sources/XCBProtocol_12_5/Build/BuildRequest.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+import MessagePack
+import XCBProtocol
+
+public struct BuildRequest {
+    public let parameters: BuildParameters
+    public let configuredTargets: [ConfiguredTarget]
+    public let continueBuildingAfterErrors: Bool
+    public let hideShellScriptEnvironment: Bool
+    public let useParallelTargets: Bool
+    public let useImplicitDependencies: Bool
+    public let useDryRun: Bool
+    public let showNonLoggedProgress: Bool
+    public let buildPlanDiagnosticsDirPath: String?
+    public let buildCommand: BuildCommand
+    public let schemeCommand: SchemeCommand
+    public let buildOnlyTheseFiles: MessagePackValue
+    public let buildOnlyTheseTargets: MessagePackValue
+    public let buildDescriptionID: MessagePackValue
+    public let enableIndexBuildArena: Bool
+    public let unknown: MessagePackValue? // comes back as `nil`, so it's unclear what this is or what type it is
```

Comment:
```suggestion
    public let unknown: MessagePackValue // comes back as `.nil`, so it's unclear what this is or what type it is
```

Reply:
Is there a new string (or strings, in case this is an object) after `enableIndexBuildArena` or before `useLegacyBuildLocations`?